### ISSUE TITLE
[Bug]: unable to change the background of the preview by using keyboard.

### ISSUE THREAD

**Issue by msftedad** [storybookjs/storybook#25805]
### Describe the bug

While expanding the "tab list" the keyboard focus moving to the "components" button.

### To Reproduce

1.open the URL " [Overview - Docs ⋅ Storybook (windows.net)](https://vivacommonstorybooktest.blob.core.windows.net/$web/index.html?path=/docs/overview--docs)".
2.Navigate to the "Viva web typography" in the left pane and expand it.
3.observe the keyboard focus after "skip to side bar".

### System

```bash
Actual Results:
After hitting tab key from the "skip to side bar" keyboard focus moving to the non inter active element.
Expected Results:	
After hitting tab key from the "skip to side bar" keyboard focus moves to the interactive element in the page.
```

### Additional context

[focus lost after tabbing from 'skip to sidebar'_edit_0 (1).webm](https://github.com/storybookjs/storybook/assets/93735775/335f7b9c-a501-4afd-b7df-ec90d0298121)
[keyboard focus moving to the non inter active element.webm](https://github.com/storybookjs/storybook/assets/93735775/a725e33d-5499-462a-aa56-40dd9ab8a244)

**Comment by maudem** [comment]
I'd like to pick this issue up

**Comment by hmcostaa** [comment]
Hi @maudem, can I fix this one? It's for a project in university.

**Comment by JReinhold** [comment]
I think this is intended behavior. The element following "Skip to sidebar" in the focus sequence is the _iframe_ that has the preview (stories/docs). That is the one being focused automatically by the browser - even though the iframe might not contain interactive elements.

If this didn't happen the user would have no way to focus the preview, which would make it impossible to scroll the preview with the keyboard when necessary.

According to HTML Standards, `iframe` is a focusable area so it _should_ get focus when appropriate. See "Example" at https://html.spec.whatwg.org/multipage/interaction.html#data-model

I'm not an expert on this but I think the focus behavior is controlled by the browser, and while we _could_ explicitly attempt to remove the iframe from the focus sequence with some heuristics, I don't think we should as that would probably lead to more severe bugs in other edge cases where we can't detect the content properly.
